eZ/Bundle/EzPublishCoreBundle/Tests/EventListener/ViewControllerListenerTest.php

```diff
@@ -151,6 +158,63 @@ public function testGetControllerMatchedView()
         $this->assertEquals($expectedView, $this->request->attributes->get('view'));
     }
 
+    public function testGetControllerEmitsProperEvents(): void
+    {
+        $viewObject = new class() extends BaseView {
+        };
+
+        $this->viewBuilderRegistry
+            ->expects($this->once())
+            ->method('getFromRegistry')
+            ->will($this->returnValue($this->viewBuilderMock));
```

Comment:
We don't need to go into phpunit mock internals :)
```suggestion
            ->willReturn($this->viewBuilderMock);
```